tools/packaging/static-build/agent/build.sh

```diff
@@ -16,13 +17,13 @@ source "${script_dir}/../../scripts/lib.sh"
 container_image="${AGENT_CONTAINER_BUILDER:-$(get_agent_image_name)}"
 [ "${CROSS_BUILD}" == "true" ] && container_image="${container_image}-cross-build"
 
-sudo docker pull ${container_image} || \
+sudo docker pull --platform=linux/${target_arch} ${container_image} || \
 	(sudo docker $BUILDX build $PLATFORM \
 	    	--build-arg RUST_TOOLCHAIN="$(get_from_kata_deps "languages.rust.meta.newest-version")" \
 		-t "${container_image}" "${script_dir}" && \
 	 # No-op unless PUSH_TO_REGISTRY is exported as "yes"
 	 push_to_registry "${container_image}")
-
+exit 1
```

Comment:
why 'exit 1' here?

Reply:
This isn't there anymore.

Reply:
Yes. But it's better to squash "+ exit 1" here with "- exit 1" in next commit. WDYT? 😄

Reply:
The exit is squashed to the second commit.